app/src/main/java/com/quran/labs/androidquran/service/AudioService.java

```diff
@@ -1111,10 +1111,11 @@ private void setUpAsForeground() {
               emptyTitles ? "" : getString(R.string.previous), previousIntent)
           .addAction(R.drawable.ic_pause, emptyTitles ? "" : getString(R.string.pause), pauseIntent)
           .addAction(R.drawable.ic_next, emptyTitles ? "" : getString(R.string.next), nextIntent)
-          .setShowWhen(false);
+          .setShowWhen(false)
```

Comment:
It's `NotificationCompat` after all. It should have handled such cases internally and say _"Oh we got `setShowWhen(false)`, but we are running on an old platform. Let's replace it with `setWhen(0)`"_. But somehow it doesn't do that :)